server.js

```diff
@@ -0,0 +1,12 @@
+// const express = require("express")
+// const app = express()
+
+// app.get("/api/data", (req, res) => {
+//     res.json({name: "Danesh Rajasolan"})
+// })
+
+// const port = process.env.PORT || 4000
+
+// app.listen(port, () => {
+//     console.log(`Server running on port ${port}...`)
+// })
```

Comment:
please don't commit commented out code

Reply:
oops overlooked this, my bad, will remove this from the commit